src/main/java/mate/academy/bookstore/repository/category/CategoryRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.bookstore.repository.category;
+
+import mate.academy.bookstore.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+public interface CategoryRepository
+        extends JpaRepository<Category, Long>, JpaSpecificationExecutor<Category> {
```

Comment:
```suggestion
        extends JpaRepository<Category, Long> {
```
Do you really  need  JpaSpecificationExecutor?